Vagrantfile

```
@@ -9,14 +9,14 @@ SCRIPT
 
 Vagrant.configure("2") do |config|
 
-    config.vm.box = "bento/ubuntu-20.04"
-    config.vm.box_version = "202012.23.0"
+    config.vm.box = "bento/ubuntu-18.04"
```

I would leave it on 20.04 because that works and then we can fix any particular issues that arise. But everything else LGTM
```suggestion
    config.vm.box = "bento/ubuntu-20.04"
```